api/data/page-filters/page-filters-csv-parser.ts

```diff
@@ -0,0 +1,91 @@
+import * as fs from 'fs';
+import { Options, parse } from 'csv-parse';
+
+const GROUP_HEADING_REGEX = /^\(.*\)$/;
+
+const isGroupHeadingRow = (row: string[]): boolean => {
```

Comment:
This CSV parsing is avoidable. Given that we didn't know how we were going to consume filters at the time of implementing this, the assumption was to mock some data temporarily to unblock FE

We could just create a SQL and insert the data adding a DROP statement. Adding a manually crafted CSV, to then parse it to generate a SQL, to then inject it is a huge redundancy that we could just avoid and get rid of a lot of unnecessary complexity